beacon-chain/operations/slashings/service.go

```diff
@@ -36,7 +38,9 @@ func (p *Pool) PendingAttesterSlashings(ctx context.Context, state *beaconstate.
 	numPendingAttesterSlashings.Set(float64(len(p.pendingAttesterSlashing)))
 
 	included := make(map[uint64]bool)
-	pending := make([]*ethpb.AttesterSlashing, 0, params.BeaconConfig().MaxAttesterSlashings)
+	// Allocate pending slice with a capacity of min(len(p.pendingAttesterSlashing), maxAttesterSlashings)
+	// since the array cannot exceed the max and is typically less than the max value.
+	pending := make([]*ethpb.AttesterSlashing, 0, mathutil.Min(uint64(len(p.pendingAttesterSlashing)), params.BeaconConfig().MaxAttesterSlashings))
```

Comment:
Shouldn't this be max? it cant go beyond `MaxAttesterSlashings` right?

Reply:
That’s true, but there’s no point in allocating extra data if there aren’t no slashing.

Reply:
You are right. For some reason my brain was too stubborn to think that...

Reply:
It's ok. I did the same thing somewhere when I wrote Max lol